contracts/0.8.25/vaults/Dashboard.sol

```diff
@@ -274,6 +487,18 @@ contract Dashboard is AccessControlEnumerable {
         vaultHub.burnStethBackedByVault(address(stakingVault), _tokens);
     }
 
+    /**
+     * @dev calculates total shares vault can mint
+     * @param _valuation custom vault valuation
+     */
+    function _totalMintableShares(uint256 _valuation) internal view returns (uint256) {
```

Comment:
I think this must be replaced for _maxMintableShares from the VaultHub... maybe we heed to make it public in the hub
```
function _maxMintableShares(address _vault, uint256 _reserveRatio) internal view returns (uint256) {
        uint256 maxStETHMinted = (IStakingVault(_vault).valuation() * (TOTAL_BASIS_POINTS - _reserveRatio)) /
            TOTAL_BASIS_POINTS;
        return STETH.getSharesByPooledEth(maxStETHMinted);
    }
```